docs/Install & Config.md

```diff
@@ -50,6 +50,10 @@ method: bundle
 #  Default is `null`.
 babel: true
 
+# Define core babel package
```

Comment:
Describe why youd ever want to specify this option. Its not clear from this MR at all why you'd need to do this.